src/spells/CESFork_GoerliRwaSpell.sol

```diff
@@ -189,14 +184,14 @@ contract SpellAction {
         JugAbstract(MCD_JUG).init(ilk);
 
         /// @notice Allow RWA007SGHWOFH1 Join to modify Vat registry
-        VatAbstract(MCD_VAT).rely(MCD_JOIN_RWA007SGFWOFH1_A);
+        VatAbstract(MCD_VAT).rely(MCD_JOIN_RWA007_A);
 
         /// @notice Allow RwaLiquidationOracle to modify Vat registry
         VatAbstract(MCD_VAT).rely(MIP21_LIQUIDATION_ORACLE);
 
         /// @notice 1000 debt ceiling
-        VatAbstract(MCD_VAT).file(ilk, "line", RWA007SGHWOFH1_A_INITIAL_DC);
-        VatAbstract(MCD_VAT).file("Line", VatAbstract(MCD_VAT).Line() + RWA007SGHWOFH1_A_INITIAL_DC);
+        VatAbstract(MCD_VAT).file(ilk, "line", RWA007_A_INITIAL_DC);
+        VatAbstract(MCD_VAT).file("Line", VatAbstract(MCD_VAT).Line() + RWA007_A_INITIAL_DC);
 
         /// @notice No dust
         // VatAbstract(MCD_VAT).file(ilk, "dust", 0)
```

Comment:
Why don't we have a value for `dust`?

Reply:
Not sure if we need that, actually this was commented in MIP21 repo too

Reply:
Yeah, I assume not. Client will likely go straight to debt ceiling, if they want to mingle with little values it does not really introduce risk, just gas spending.